src/test/e2e/tests/specs.test.ts

```diff
@@ -37,6 +38,50 @@ let caseReferenceNumberSearchPage: CaseReferenceNumberSearchPage;
 let caseReferenceNumberSearchResultPage: CaseReferenceNumberSearchResultsPage;
 let courtNameSearchPage: CourtNameSearchPage;
 let caseEventGlossaryPage: CaseEventGlossaryPage;
+const signInPage = new SignInPage;
+
+describe('Sign In Page', () => {
```

Comment:
please make sure you put these in the right places, we have a verfified user journey, this signing in is the start of that process, you can also remove the tests that test google

Reply:
we need to test an external link therefore as placeholder is google until we have the final url

Reply:
I will move the test in the verified user section